backend/vehicle/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.contrib import admin
 from django.urls import include, path
 
-from .views import VehicleDetailApiView, VehicleListApiView
+from .views import VehicleDetailApiView, VehicleListApiView, VehicleFilterList
 
 urlpatterns = [
-    path("", VehicleListApiView.as_view(), name="vehicle"),
+    path("create/", VehicleListApiView.as_view(), name="vehicle"),
```

Comment:
@kevinrczhang We can remove this. Calling POST on the root endpoint will already create the vehicle (which is the convention for writing endpoints).

Reply:
will do